### PR TITLE
Add support for double type

### PR DESCRIPTION
**Pull request by stephanvd**

### CODE REVIEW

**Comment by jheth** [comment]
@stephanvd Sorry for the slow response to these PRs.  Can you rebase this branch from master?

File: lib/dynamics_crm/xml/attributes.rb [hunk 1]
```diff
@@ -33,6 +33,8 @@ def get_type(key, value)
             type = "FetchExpression"
           when Money
             type = "Money"
+          when DynamicsCRM::Metadata::Double
```
Comment (jheth) [inline lib/dynamics_crm/xml/attributes.rb]: Why is a separate class type necessary?  The rest of the primitives are handled without a custom class.

Comment (stephanvd) [inline lib/dynamics_crm/xml/attributes.rb]: Ruby doesn't have a double type. BigDecimal/Float are mapped to decimal in CRM already, so this seemed the way to go.

**Comment by jheth** [comment]
Thanks @stephanvd . I'm good with this now.  I'm not sure if Double will remain under the Metadata namespace long term but I like the added support.

**Pull request merged**